build.gradle.kts

```diff
@@ -0,0 +1,157 @@
+import org.jetbrains.intellij.tasks.RunPluginVerifierTask
+import java.util.EnumSet
+
+group = "com.squareup.cash.hermit"
+version = project.properties["version"] ?: "1.0-SNAPSHOT"
+
+plugins {
+  id("idea")
+  id("java")
+  kotlin("kapt") version "1.8.10"
+  id("org.jetbrains.intellij") version "1.13.3"
+  id("org.jetbrains.kotlin.jvm") version "1.8.10"
+  id("org.jetbrains.kotlin.plugin.serialization") version "1.4.32"
+}
+
+// region Build, dependencies
+val kotlin_version = "1.8.10"
```

Comment:
Is this duplicating the version number in the `plugins` block above?

Reply:
ah yes

Reply:
I've just tried reusing it. There's some funky stuff with scope in the `plugins {...}` block so it's not straightforward. I'll leave as is.